jsonapi/cashout.go

```diff
@@ -0,0 +1,17 @@
+package jsonapi
+
+type Cashout struct {
+	Name  string `json:"name"`
+	Id    string `json:"id"`
+	Agent struct {
+		Meta struct {
+			Href string `json:"href"`
+		} `json:"meta"`
+	} `json:"agent"`
+	Project struct {
+		Meta struct {
+			Href string `json:"href"`
+		} `json:"meta"`
+	} `json:"project"`
+	Description string `json:"description"`
+}
```

Comment:
Пусть кэшаут тоже удовлетворяет интерфейсу если уж у него есть name и id
```suggestion
}

func (c Cashout) GetName() string {
	return c.Name
}

func (c Cashout) GetID() string {
	return c.Id
}
```